website/siteConfig.js

```diff
@@ -0,0 +1,64 @@
+// See https://docusaurus.io/docs/site-config.html for all the possible
+// site configuration options.
+
+const repoUrl = "https://github.com/scalacenter/scalafix";
+
+const siteConfig = {
+  title: "Scalafix",
+  tagline: "Refactoring and linting tool for Scala",
+  url: "https://scalacenter.github.io/",
+  baseUrl: "/scalafix/",
+
+  // Used for publishing and more
+  projectName: "scalafix",
+  organizationName: "scalacenter",
+
+  algolia: {
+    apiKey: "???",
```

Comment:
you may want to remove the `algolia` section until you have a valid API, so that the search bar doesn't appear